hypershift-operator/controllers/nodepool/kubevirt/kubevirt_test.go

```diff
@@ -38,10 +39,11 @@ const (
 
 func TestKubevirtMachineTemplate(t *testing.T) {
 	testCases := []struct {
-		name     string
-		nodePool *hyperv1.NodePool
-		hcluster *hyperv1.HostedCluster
-		expected *capikubevirt.KubevirtMachineTemplateSpec
+		name                    string
+		nodePool                *hyperv1.NodePool
+		hcluster                *hyperv1.HostedCluster
+		expected                *capikubevirt.KubevirtMachineTemplateSpec
+		expectedValidationError types.GomegaMatcher
```

Comment:
nit: why do you pass the matcher and not the error ? 

AFAIU, you'll only match error (especially given the attribute's name).

This is really a nit, but it caught my eye.

Reply:
To give more flexibility in case we want to check substrings or the like.